Sources/Networking/Backend.swift

```diff
@@ -14,16 +14,16 @@
 import Foundation
 
 typealias SubscriberAttributeDict = [String: SubscriberAttribute]
-typealias BackendCustomerInfoResponseHandler = (Result<CustomerInfo, Error>) -> Void
-typealias IntroEligibilityResponseHandler = ([String: IntroEligibility], Error?) -> Void
-typealias OfferingsResponseHandler = (Result<[String: Any], Error>) -> Void
-typealias OfferSigningResponseHandler = (Result<PostOfferForSigningOperation.SigningData, Error>) -> Void
-typealias SimpleResponseHandler = (Error?) -> Void
-typealias LogInResponseHandler = (Result<(info: CustomerInfo, created: Bool), Error>) -> Void
 
 class Backend {
 
-    static let RCSuccessfullySyncedKey: NSError.UserInfoKey = "rc_successfullySynced"
```

Comment:
This is gone in favor of `NetworkError.successfullySynced`

Reply:
YESSSSSSssss